depends/packages/libpng.mk

```
@@ -0,0 +1,24 @@
+package=libpng
+$(package)_version=1.6.37
+$(package)_download_path=https://downloads.sourceforge.net/project/libpng/libpng16/$($(package)_version)/
+$(package)_file_name=$(package)-$($(package)_version).tar.gz
+$(package)_sha256_hash=c509d15ebdbfa355469828df2edcba15c5656761dd3037fcf28c206b5268a035
```

Switch to LZMA-compressed archive?
```suggestion
$(package)_file_name=$(package)-$($(package)_version).tar.xz
$(package)_sha256_hash=505e70834d35383537b6491e7ae8641f1a4bed1876dbfe361201fc80868d88ca
```

less by 50% :)